.github/ISSUE_TEMPLATE/ideas.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Ideas
+about: Have a new idea/feature for the bot?
```

Comment:
Maybe we can use "for the project?"
I thin this isn't limited to a bot(and ideas may vary and be about stuff not related to the bot)

Reply:
Okay - I'll change that